### PR TITLE
Test/ui tests

### PR DESCRIPTION
**Pull request by LarsHoldijk**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-sudo: false
```
Comment (FrankTN) [inline .travis.yml:1]: This is extremely smart, who did this?

Comment (Feikoritsema) [inline .travis.yml:1]: Not so smart

File: Hestia/app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -64,7 +64,7 @@
     <string name="serverFound">Server Found: </string>
     <string name="findServerTitle">"Find server"</string>
     <string name="setIpHint">Enter ip here</string>
-    <string name="versionToLow">Android version to low</string>
+    <string name="versionTooLow">Android version too low</string>
```
Comment (FrankTN) [inline Hestia/app/src/main/res/values/strings.xml:67]: Nice

**Review by FrankTN — APPROVED** [review]
Looks good

**Pull request merged**